documentation/user-interface/report-view.md

```diff
@@ -68,7 +68,7 @@ Click the ![Screenshot Icon](../../images/user-interface/report-view/screenshot-
 
 #### Examine Failed Tests
 
-If a test has failed, an arrow is displayed near its status. This arrow allows you to expand the grid row and see details about this fail.
+An arrow is displayed near a test's status to indicate if it failed. This arrow allows you to expand the grid row and see details about this failure.

```

Comment:
An arrow does not actually indicate that a test failed. We have an icon for that. The arrow just allows you to expand the row and see the detailed info.